router/batchrouter/handle_async.go

```diff
@@ -452,7 +462,14 @@ func (brt *Handle) sendJobsToStorage(batchJobs BatchedJobs) {
 			out.SuccessResponse = fmt.Sprintf(`{"error":"%s"`, rterror.DisabledEgress.Error()) // skipcq: GO-R4002
 		}
 
-		brt.setMultipleJobStatus(out, false, getAttemptNumbers(batchJobs.Jobs), getFirstAttemptAts(batchJobs.Jobs), getOriginalJobParameters(batchJobs.Jobs))
+		brt.setMultipleJobStatus(SetMultipleJobStatusParams{
+			AsyncOutput:           out,
+			Attempted:             false,
```

Comment:
no need to pass false values as go default false anyway

Reply:
Addressed